
<!--==========================================================================
	FTL macros 
	========================================================================== -->


<!--==========================================================================
	CSS styles 
	========================================================================== -->


<style type="text/css">

body {
	background-color: #dddddd;
}

/* Table styles. */

table {
  border-color: #000;
  border-spacing: 0px;
  border-style: solid;
  border-width: 1px;
  cell-spacing: 0px;
}

.noBorder {
  border-width: 0px;
}

td, th {
  font-family: Arial, Helvetica, sans-serif;
  font-size: 10pt;
  padding: 2px 0.5em;
  white-space: nowrap;
}

td.numeric {
  text-align: right;
}

th {
  background-color: #c0c0c0;
}

th.mainHeader {
  background-color: #808080;
  color: #ffffff;
  text-align: left;
}

th a {
  color: #000080;
  text-decoration: none;
}

th a:visited {
  color: #000080;
}

th a:active, th a:hover {
  color: #800000;
  text-decoration: underline;
}

.toc {
	border: 1px solid #aaa;
	background-color: #eeeeee;
	padding: 5px;
	font-size: 95%;
}

.note {
	border: 1px solid #aaa;
	background-color: #eeeeee;
	padding: 5px;
	font-size: 90%;
	text-align: left;
}

.main {
	margin-top: 15px;
	width: 1000px;
	text-align: left;
	background: #ffffff;
	border: 3px solid #777777;
}

.histo {
	table-layout:fixed;
	width:100%;
	border:1px solid #aaa;
	word-wrap:break-word;
}

</style>


<!--==========================================================================
	Index
	========================================================================== -->

<center>
<div class="main">

<center> <h3> SnpEff: Variant analysis </h3> </center>

<div style="margin-left: .5em">
<table class="toc"><tr><td>
	<center><b>Contents</b></center>
	<a href="#summary">Summary</a><br>
	<a href="#changeRateChr"> Change rate by chromosome</a><br>
	<a href="#changesByType">Variants by type</a><br>
	<a href="#effectsImpact"> Number of variants by impact </a><br>
	<a href="#effectsImpact"> Number of variants by functional class </a><br>
	<a href="#effects"> Number of variants by effect </a><br>
	<a href="#quality">Quality histogram</a><br>
	<a href="#coverage">Coverage histogram</a><br>
	<a href="#baseChages">Base change table</a><br>
	<a href="#tstv">Transition vs transversions (ts/tv)</a><br>
	<a href="#alleleFreq"> Frequency of alleles </a><br>
	<a href="#codonChanges"> Codon change table </a><br>
	<a href="#aaChanges"> Amino acid change table </a><br>
	<a href="#chrChanges"> Chromosome change plots </a><br>
	<a href="snpEff_genes.txt"> Details by gene </a><br>
</tr></td></table>
</div>


<!--==========================================================================
	Summary table 
	========================================================================== -->

<hr>
<a name="summary">
<center>
<b>Summary</b><p>

<table border=0>
	<tr bgcolor=ffffff> 
		<td valign=top> <b> Genome </b> </td>
		<td> GRCm37.67 </td>
	</tr>
	<tr bgcolor=dddddd> 
		<td valign=top> <b> Date </b> </td>
		<td> 2016-10-02 02:57 </td>
	</tr>
	<tr bgcolor=ffffff> 
		<td valign=top> <b> SnpEff version </b> </td>
		<td> <pre>SnpEff 3.6c (build 2014-05-20), by Pablo Cingolani</pre> </td>
	</tr>
	<tr bgcolor=dddddd> 
		<td valign=top> <b> Command line arguments </b> </td>
		<td> <pre>SnpEff  GRCm37.67 ./aln-1449_sort_fb-nf.vcf 
</pre> </td>
	</tr>
	<tr bgcolor=ffffff> 
		<td valign=top> <b> Warnings </b> </td>
		   
		<td bgcolor="#ff0000"> 1 </td>
	</tr>	
	<tr bgcolor=dddddd>
		<td valign=top> <b> Number of lines (input file) </b> </td>
		<td> 74 </td>
	</tr>
	<tr bgcolor=ffffff>
		<td valign=top> <b> Number of variants (before filter) </b> </td>
		<td> 74 </td>
	</tr>
	<tr bgcolor=dddddd>
		<td valign=top> <b> Filter </b> </td>
		<td>  </td>
	</tr>
	<tr bgcolor=dddddd>
		<td valign=top> <b> Number of variants filtered out </b> </td>
		<td> 0 </td>
	</tr>
	<tr bgcolor=ffffff>
		<td valign=top> <b> Number of not variants <br>(i.e. reference equals alternative) </b> </td>
		<td> 0 </td>
	</tr>
	<tr bgcolor=dddddd>
		<td valign=top> <b> Number of variants processed <br> (i.e. after filter and non-variants) </b> </td>
		<td> 74 </td>
	</tr>
	<tr bgcolor=ffffff>
		<td valign=top> <b> Number of known variants <br>(i.e. non-empty ID) </b> </td>
		<td> 
		0
		( 0% ) 
		</td>
	</tr>
	<tr bgcolor=ffffff>
		<td valign=top> <b> Number of effects </b> </td>
		<td> 96 </td>
	</tr>
	<tr bgcolor=dddddd>
		<td valign=top> <b> Genome total length </b> </td>
		<td> 2,716,965,481 </td>
	</tr>
	<tr bgcolor=ffffff>
		<td valign=top> <b> Genome effective length </b> </td>
		<td> 429,998,529 </td>
	</tr>
	<tr bgcolor=dddddd>
		<td valign=top> <b> Change rate </b> </td>
		<td> 1 change every 5,810,790 bases </td>
	</tr>
</table>
<p>
</center>

<!--==========================================================================
	Change rate by chromosome
	========================================================================== -->

<hr> 
<a name="changeRateChr"> 

<center>
<b> Change rate details </b><p>

<table border=1>
	<tr><th> Chromosome </th><th> Length </th><th> Changes </th><th> Change rate </th></tr> 
		<tr>
			<td> 4 </td>
			<td class="numeric"> 155,630,120 </td>
			<td class="numeric"> 22 </td>
			<td class="numeric"> 7,074,096 </td>
		</tr> 
		<tr>
			<td> 7 </td>
			<td class="numeric"> 152,524,553 </td>
			<td class="numeric"> 17 </td>
			<td class="numeric"> 8,972,032 </td>
		</tr> 
		<tr>
			<td> 11 </td>
			<td class="numeric"> 121,843,856 </td>
			<td class="numeric"> 35 </td>
			<td class="numeric"> 3,481,253 </td>
		</tr> 
	<tr>
		<th> Total </th>
		<th class="numeric"> 429,998,529 </th>
		<th class="numeric"> 74 </th>
		<th class="numeric"> 5,810,790 </th>
	</tr> 
</table> 
</center>

<!--==========================================================================
	Changes by type
	========================================================================== -->

<hr> 
<a name="changesByType"> 
<center>
<b> Number changes by type</b><p>

<table border=1>
	<thead>
		<tr>
			<th> <b> Type   </b> </th>
			<th> <b> Total  </b> </th>
			<th> <b> Homo   </b> </th>
			<th> <b> Hetero </b> </th>
		</tr>
	</thead>
    <tbody>
	    <tr>
	    	<td> <b> SNP </b> </td>
	    	<td class="numeric" bgcolor="#ff0000"> 58 </td>
	    	<td class="numeric" bgcolor="#ff0000"> 44 </td>
	    	<td class="numeric" bgcolor="#ff0000"> 14 </td>
	    </tr>
	    <tr>
	    	<td> <b> MNP </b> </td>
	    	<td class="numeric" bgcolor="#1ae400"> 7 </td>
	    	<td class="numeric" bgcolor="#23db00"> 7 </td>
	    	<td class="numeric" bgcolor="ffffff"> 0 </td>
	    </tr>
	    <tr>
	    	<td> <b> INS </b> </td>
	    	<td class="numeric" bgcolor="#00ff00"> 1 </td>
	    	<td class="numeric" bgcolor="#00ff00"> 1 </td>
	    	<td class="numeric" bgcolor="ffffff"> 0 </td>
	    </tr>
	    <tr>
	    	<td> <b> DEL </b> </td>
	    	<td class="numeric" bgcolor="#11ed00"> 5 </td>
	    	<td class="numeric" bgcolor="#11ed00"> 4 </td>
	    	<td class="numeric" bgcolor="#00ff00"> 1 </td>
	    </tr>
	    <tr>
	    	<td> <b> MIXED </b> </td>
	    	<td class="numeric" bgcolor="#08f600"> 3 </td>
	    	<td class="numeric" bgcolor="#0bf300"> 3 </td>
	    	<td class="numeric" bgcolor="ffffff"> 0 </td>
	    </tr>
	    <tr>
	    	<td> <b> Interval </b> </td>
	    	<td class="numeric" bgcolor="ffffff"> 0 </td>
	    	<td class="numeric" bgcolor="ffffff"> 0 </td>
	    	<td class="numeric" bgcolor="ffffff"> 0 </td>
	    </tr>
    </tbody>
    <tfoot>
	    <tr>
	    	<th><b>Total </b> </th>
	    	<th class="numeric"> 74 </th>
	    	<th class="numeric"> 59 </th>
	    	<th class="numeric"> 15 </th>
	    </tr>
    </tfoot>
</table>
</center>

<!--==========================================================================
	Effects by impact
	========================================================================== -->

<hr> 
<a name="effectsImpact"> 
<center>
<b> Number of effects by impact </b> <p> 

<table border=0>
	<thead>
		<tr> 
			<th><b> Type (alphabetical order)  </b></th>
			<th> &nbsp; </th>
			<th> Count </th>
			<th> Percent </th>
		</tr>
	</thead>
		<tr> 
			<td> <b> MODERATE </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#00ff00"> 1 </td> 
			<td class="numeric" bgcolor="#00ff00"> 1.042% </td>
		</tr>
		<tr> 
			<td> <b> MODIFIER </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#ff0000"> 95 </td> 
			<td class="numeric" bgcolor="#ff0000"> 98.958% </td>
		</tr>
</table><br>
<p>
</center>

<!--==========================================================================
	Effects by functional class
	========================================================================== -->

<hr> 
<a name="effectsFuncClass"> 
<center>
<b> Number of effects by functional class </b> <p> 

<table border=0>
	<thead>
		<tr> 
			<th><b> Type (alphabetical order)  </b></th>
			<th> &nbsp; </th>
			<th> Count </th>
			<th> Percent </th>
		</tr>
	</thead>
		<tr> 
			<td> <b> MISSENSE </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="ffffff"> 1 </td> 
			<td class="numeric" bgcolor="ffffff"> 100% </td>
		</tr>
</table><br>
<p>

Missense / Silent ratio: </th><td class="numeric"> âˆž

</center>

<!--==========================================================================
	Effects stats
	========================================================================== -->

<hr> 
<a name="effects"> 
<center>
<b> Number of effects by type and region </b> <p> 

<table border=0>
	<tr>
		<th> Type </th>
		<th> Region </th>
	</tr>
	<tr>
		<td> <table border=0>
	<thead>
		<tr> 
			<th><b> Type (alphabetical order)  </b></th>
			<th> &nbsp; </th>
			<th> Count </th>
			<th> Percent </th>
		</tr>
	</thead>
		<tr> 
			<td> <b> DOWNSTREAM </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#39c500"> 12 </td> 
			<td class="numeric" bgcolor="#39c500"> 12.5% </td>
		</tr>
		<tr> 
			<td> <b> INTERGENIC </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#48b600"> 15 </td> 
			<td class="numeric" bgcolor="#48b600"> 15.625% </td>
		</tr>
		<tr> 
			<td> <b> INTRAGENIC </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#1ae400"> 6 </td> 
			<td class="numeric" bgcolor="#1ae400"> 6.25% </td>
		</tr>
		<tr> 
			<td> <b> INTRON </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#ff0000"> 50 </td> 
			<td class="numeric" bgcolor="#ff0000"> 52.083% </td>
		</tr>
		<tr> 
			<td> <b> NON_SYNONYMOUS_CODING </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#00ff00"> 1 </td> 
			<td class="numeric" bgcolor="#00ff00"> 1.042% </td>
		</tr>
		<tr> 
			<td> <b> UPSTREAM </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#2ed000"> 10 </td> 
			<td class="numeric" bgcolor="#2ed000"> 10.417% </td>
		</tr>
		<tr> 
			<td> <b> UTR_3_PRIME </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#05f900"> 2 </td> 
			<td class="numeric" bgcolor="#05f900"> 2.083% </td>
		</tr>
</table><br>
 </td>
		<td> <table border=0>
	<thead>
		<tr> 
			<th><b> Type (alphabetical order)  </b></th>
			<th> &nbsp; </th>
			<th> Count </th>
			<th> Percent </th>
		</tr>
	</thead>
		<tr> 
			<td> <b> DOWNSTREAM </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#39c500"> 12 </td> 
			<td class="numeric" bgcolor="#39c500"> 12.5% </td>
		</tr>
		<tr> 
			<td> <b> EXON </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#00ff00"> 1 </td> 
			<td class="numeric" bgcolor="#00ff00"> 1.042% </td>
		</tr>
		<tr> 
			<td> <b> INTERGENIC </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#48b600"> 15 </td> 
			<td class="numeric" bgcolor="#48b600"> 15.625% </td>
		</tr>
		<tr> 
			<td> <b> INTRON </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#ff0000"> 50 </td> 
			<td class="numeric" bgcolor="#ff0000"> 52.083% </td>
		</tr>
		<tr> 
			<td> <b> NONE </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#1ae400"> 6 </td> 
			<td class="numeric" bgcolor="#1ae400"> 6.25% </td>
		</tr>
		<tr> 
			<td> <b> UPSTREAM </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#2ed000"> 10 </td> 
			<td class="numeric" bgcolor="#2ed000"> 10.417% </td>
		</tr>
		<tr> 
			<td> <b> UTR_3_PRIME </b> </td> 
			<th> &nbsp; </th>
			<td class="numeric" bgcolor="#05f900"> 2 </td> 
			<td class="numeric" bgcolor="#05f900"> 2.083% </td>
		</tr>
</table><br>
 </td>
	</tr>
</table>
<p>

<img src="http://chart.apis.google.com/chart?cht=bvs&chbh=50,4,8&chs=800x300&chxt=y,y,x,x&chtt=Variations&chxp=1,50.0|3,50.0&chxr=0,0.0,52.083333333333336|1,0.0,100.0|3,0.0,100.0&chco=00FFFF,FA8072,FF0000,A52A2A,FFA500,DEB887,E9967A,7FFFD4,808080&chdl=Upstream|5%27UTR|Exon|Splice+Donor|Intron|Splice+Acceptor|3%27UTR|Downstream|Intergenic&chxl=1:|%25|2:|Intergenic|Up|5%27UTR|Exon|Donor|Intron|Acceptor|Exon|3%27UTR|Down|Intergenic|3:|&chd=e:AAMzAAAAAAAAAAAAAAAAAA,AAAAAAAAAAAAAAAAAAAAAA,AAAAAABSAAAAAABSAAAAAA,AAAAAAAAAAAAAAAAAAAAAA,AAAAAAAAAA..AAAAAAAAAA,AAAAAAAAAAAAAAAAAAAAAA,AAAAAAAAAAAAAAAACkAAAA,AAAAAAAAAAAAAAAAAAPXAA,TNAAAAAAAAAAAAAAAAAATN" border=1><p>

</center>
<!--==========================================================================
	Quality plots
	========================================================================== -->

	<hr> 
	<a name="quality"> 
	<b> Quality: </b> 
	<p>
	<pre>
	<table class="histo">
		<tr> <th width=15%>Min</th><td>0</td> </tr>
		<tr> <th>Max</th><td>3,441</td> </tr>
		<tr> <th>Mean</th><td>70.892</td> </tr>
		<tr> <th>Median</th><td>2</td> </tr>
		<tr> <th>Standard deviation</th><td>400.708</td> </tr>
		<tr> <th>Values</th><td>0,1,3,4,8,11,12,15,16,18,21,25,27,28,37,46,48,55,57,58,64,68,98,104,109,112,292,300,3441</td> </tr>
		<tr> <th>Count</th><td>34,3,3,2,1,1,2,2,3,1,3,1,1,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1</td> </tr>
		</tr>
	</table>
		<img src="http://chart.apis.google.com/chart?cht=bvg&chbh=8,4,2&chs=800x300&chxt=y,y,x,x&chg=344.0,3.0,3,2&chxp=1,50.0|3,50.0&chtt=Quality+histogram&chxr=0,1.0,34.0|1,0.0,100.0|3,0.0,100.0&chd=e:..D2D2B7AAAAB7B7D2AAD2AAAAAAAAAAAAB7AAAAAAAAAAAAAAAAAAAAAA&chxl=1:|Count|2:|0||||||||||21||||||||||64||||||||3441|3:|Quality"><br>
	</pre>

<!--==========================================================================
	Coverage
	========================================================================== -->

	<hr> 
	<a name="coverage"> 
	<b> Coverage:</b> 
	<p>
	<pre>
	<table class="histo">
		<tr> <th width=15%>Min</th><td>2</td> </tr>
		<tr> <th>Max</th><td>121</td> </tr>
		<tr> <th>Mean</th><td>6.392</td> </tr>
		<tr> <th>Median</th><td>3</td> </tr>
		<tr> <th>Standard deviation</th><td>14.338</td> </tr>
		<tr> <th>Values</th><td>2,3,4,5,6,7,8,9,12,14,15,17,29,121</td> </tr>
		<tr> <th>Count</th><td>31,9,13,7,1,1,1,1,1,4,1,2,1,1</td> </tr>
		</tr>
	</table>
		<img src="http://chart.apis.google.com/chart?cht=bvg&chbh=8,4,2&chs=800x300&chxt=y,y,x,x&chg=11.0,3.0,3,2&chxp=1,50.0|3,50.0&chtt=Coverage+histogram&chd=e:..QpZmMzAAAAAAAAAAGaAAB7AAAA&chxr=0,1.0,31.0|1,0.0,100.0|3,0.0,100.0&chxl=1:|Count|2:|2||||||||||15|||121|3:|Coverage"><br>
	</pre>

<!--==========================================================================
	InDels
	========================================================================== -->

	<hr> 
	<b> Insertions and deletions length:</b> 
	<p>
	<pre>
	<table class="histo">
		<tr> <th width=15%>Min</th><td>-3</td> </tr>
		<tr> <th>Max</th><td>2</td> </tr>
		<tr> <th>Mean</th><td>-1</td> </tr>
		<tr> <th>Median</th><td>-1</td> </tr>
		<tr> <th>Standard deviation</th><td>1.673</td> </tr>
		<tr> <th>Values</th><td>-3,-2,-1,2</td> </tr>
		<tr> <th>Count</th><td>1,1,3,1</td> </tr>
		</tr>
	</table>
		<img src="http://chart.apis.google.com/chart?cht=bvg&chbh=8,4,2&chs=800x300&chxt=y,y,x,x&chd=e:AAAA..AA&chxp=1,50.0|3,50.0&chxl=1:|Count|2:|-3|||2|3:|Length&chxr=0,1.0,3.0|1,0.0,100.0|3,0.0,100.0&chtt=Insertion+deletion+length+histogram"><br>
	</pre>

<!--==========================================================================
	Base changes
	========================================================================== -->

<hr> 
<a name="baseChages"> 
<center>
<b> Base changes (SNPs) </b> <p>

<table border=1>
    <tr>
    	<td> &nbsp; </td>  <th> <b> A </b> </th> <th> <b> C </b> </th> <th> <b> G </b> </th> <th> <b> T </b> </th>
    </tr>
	    <tr> <th> <b> A </b> </th><td class="numeric" bgcolor="ffffff"> 0 </td><td class="numeric" bgcolor="#5ba300"> 6 </td><td class="numeric" bgcolor="#36c800"> 4 </td><td class="numeric" bgcolor="#00ff00"> 1 </td> </tr>
	    <tr> <th> <b> C </b> </th><td class="numeric" bgcolor="#5ba300"> 6 </td><td class="numeric" bgcolor="ffffff"> 0 </td><td class="numeric" bgcolor="ffffff"> 0 </td><td class="numeric" bgcolor="#48b600"> 5 </td> </tr>
	    <tr> <th> <b> G </b> </th><td class="numeric" bgcolor="#12ec00"> 2 </td><td class="numeric" bgcolor="#24da00"> 3 </td><td class="numeric" bgcolor="ffffff"> 0 </td><td class="numeric" bgcolor="#36c800"> 4 </td> </tr>
	    <tr> <th> <b> T </b> </th><td class="numeric" bgcolor="#36c800"> 4 </td><td class="numeric" bgcolor="#7f7f00"> 8 </td><td class="numeric" bgcolor="#ff0000"> 15 </td><td class="numeric" bgcolor="ffffff"> 0 </td> </tr>
</table>
</center>

<p>

<!--==========================================================================
	Ts/Tv
	========================================================================== -->

<hr> 
<a name="tstv">
<center> <b> Ts/Tv (transitions / transversions) </b> <p> </center>

<small>
<b>Note:</b> Only SNPs are used for this statistic.<br>
<b>Note:</b> This Ts/Tv ratio is a 'raw' ratio. Some people prefer to use a ratio of rates, not observed events. In that case, you need to multiply by 2.0 (since there are twice as many possible transitions than transversions, E[Ts/Tv] ratio is twice the ratio of events).
</small>
<p>

<center>
<table border=1>
	<tr> <th> Transitions </th><td class="numeric"> 19 </td> </tr>
	<tr> <th> Transversions </th><td class="numeric"> 39 </td> </tr>
	<tr> <th> Ts/Tv ratio </th><td class="numeric"> 0.4872 </td> </tr>
</table>
</center>
<p>

	<b>All variants:</b>
	<pre>Sample ,bar-1449,Total
Transitions ,26,26
Transversions ,50,50
Ts/Tv ,0.520,0.520
</pre>
	<p>
	<b>Only known variants</b> (i.e. the ones having a non-empty ID field):
	<pre>No results available (empty input?)</pre>
<p>

<!--==========================================================================
	Allele frequency
	========================================================================== -->

<hr> 
<a name="alleleFreq">
<center> <b> Frequency of alleles </b> <p> </center>

Note: Number of times an allele appears once (singleton), twice (doubletons), etc.<p>

	<b>All variants:</b>
	<img src="http://chart.apis.google.com/chart?cht=bvg&chbh=8,4,2&chd=e:AA..&chs=800x300&chxt=y,y,x,x&chxp=1,50.0|3,50.0&chtt=Alleles+frequencies&chxl=1:|Count|2:|1|2|3:|Alleles&chxr=0,14.0,31.0|1,0.0,100.0|3,0.0,100.0"><br>
	<table class="histo">
		<tr> <th width=15%>Min</th><td>1</td> </tr>
		<tr> <th>Max</th><td>2</td> </tr>
		<tr> <th>Mean</th><td>1.689</td> </tr>
		<tr> <th>Median</th><td>2</td> </tr>
		<tr> <th>Standard deviation</th><td>0.468</td> </tr>
		<tr> <th>Values</th><td>1,2</td> </tr>
		<tr> <th>Count</th><td>14,31</td> </tr>
		</tr>
	</table>
	<p>
	<b>Only known variants</b> (i.e. the ones having a non-empty ID field):
<p>


<!--==========================================================================
	Codon change table
	========================================================================== -->
	
<hr> 
<a name="codonChanges">
<center>
<b> Codon changes</b> <p>

	<div class="note">
		How to read this table: <br>
		- Rows are reference codons and columns are changed codons. E.g. Row 'AAA' column 'TAA' indicates how many 'AAA' codons have been replaced by 'TAA' codons.<br>
		- Red background colors indicate that more changes happened (heat-map).<br>
		- Diagonals are indicated using grey background color <br> 
		- WARNING: This table may include different translation codon tables (e.g. mamalian DNA and mitochondrial DNA).<br>
		<p>
	</div><p>

  
<table border=1>
	<thead>
		<tr> 
			<th> &nbsp; </th>
			 <th> ACC </th>  <th> ATC </th> 
		</tr>
	</thead>
			<tr> 
				<th> ACC </th>
					
						<th class="numeric"> &nbsp </th>
					
						<td class="numeric" bgcolor="ffffff"> &nbsp </td>
			</tr>
			<tr> 
				<th> ATC </th>
					
						<td class="numeric" bgcolor="ffffff"> 1 </td>
					
						<th class="numeric"> &nbsp </th>
			</tr>
</table>
</center>

<!--==========================================================================
	Amino acid change table
	========================================================================== -->

<hr> 
<a name="aaChanges">
<center>
<b> Amino acid changes</b><p> 

	<div class="note">
		How to read this table: <br>
		- Rows are reference amino acids and columns are changed amino acids. E.g. Row 'A' column 'E' indicates how many 'A' amino acids have been replaced by 'E' amino acids.<br>
		- Red background colors indicate that more changes happened (heat-map).<br>
		- Diagonals are indicated using grey background color <br> 
		- WARNING: This table may include different translation codon tables (e.g. mamalian DNA and mitochondrial DNA).<br>
		<p>
	</div><p>

<table border=1>
	<thead>
		<tr> 
			<th> &nbsp; </th>
			 <th> I </th>  <th> T </th> 
		</tr>
	</thead>
			<tr> 
				<th> I </th>
					
						<th class="numeric"> &nbsp </th>
					
						<td class="numeric" bgcolor="ffffff"> 1 </td>
			</tr>
			<tr> 
				<th> T </th>
					
						<td class="numeric" bgcolor="ffffff"> &nbsp </td>
					
						<th class="numeric"> &nbsp </th>
			</tr>
</table>
</center>

<!--==========================================================================
	Chromosome change table
	========================================================================== -->

<hr> 
<a name="chrChanges">
<center>
<b> Changes by chromosome</b><p> 

<center>
	<pre>
		<img src="http://chart.apis.google.com/chart?cht=lc&chs=800x300&chxt=y,y,x,x&chxp=1,50.0|3,50.0&chtt=Changes+histogram%3A+4&chxr=0,0.0,7.0|1,0.0,100.0|3,0.0,100.0&chd=e:AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA..AAAAAAAAAAAAAAAAAAAAAAAAAAkeAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA&chxl=1:|Changes%2F1000Kb|2:|500000||||||||||20500000||||||||||39500000||||||||||59500000||||||||||78500000||||||||||98500000||||||||||117500000||||||||||137500000|||||||||155000000|3:|Position"><br>
		4, Position,0,1000000,2000000,3000000,4000000,5000000,6000000,7000000,8000000,9000000,10000000,11000000,12000000,13000000,14000000,15000000,16000000,17000000,18000000,19000000,20000000,21000000,22000000,23000000,24000000,25000000,26000000,27000000,28000000,29000000,30000000,31000000,32000000,33000000,34000000,35000000,36000000,37000000,38000000,39000000,40000000,41000000,42000000,43000000,44000000,45000000,46000000,47000000,48000000,49000000,50000000,51000000,52000000,53000000,54000000,55000000,56000000,57000000,58000000,59000000,60000000,61000000,62000000,63000000,64000000,65000000,66000000,67000000,68000000,69000000,70000000,71000000,72000000,73000000,74000000,75000000,76000000,77000000,78000000,79000000,80000000,81000000,82000000,83000000,84000000,85000000,86000000,87000000,88000000,89000000,90000000,91000000,92000000,93000000,94000000,95000000,96000000,97000000,98000000,99000000,100000000,101000000,102000000,103000000,104000000,105000000,106000000,107000000,108000000,109000000,110000000,111000000,112000000,113000000,114000000,115000000,116000000,117000000,118000000,119000000,120000000,121000000,122000000,123000000,124000000,125000000,126000000,127000000,128000000,129000000,130000000,131000000,132000000,133000000,134000000,135000000,136000000,137000000,138000000,139000000,140000000,141000000,142000000,143000000,144000000,145000000,146000000,147000000,148000000,149000000,150000000,151000000,152000000,153000000,154000000,155000000
4,Count,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,14,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,8,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0

	</pre>
	<pre>
		<img src="http://chart.apis.google.com/chart?cht=lc&chs=800x300&chxt=y,y,x,x&chxp=1,50.0|3,50.0&chtt=Changes+histogram%3A+7&chxr=0,0.0,7.0|1,0.0,100.0|3,0.0,100.0&chd=e:AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAI9AAAA..AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA&chxl=1:|Changes%2F1000Kb|2:|500000||||||||||20500000||||||||||39500000||||||||||58500000||||||||||77500000||||||||||96500000||||||||||115500000||||||||||134500000|||||||||152000000|3:|Position"><br>
		7, Position,0,1000000,2000000,3000000,4000000,5000000,6000000,7000000,8000000,9000000,10000000,11000000,12000000,13000000,14000000,15000000,16000000,17000000,18000000,19000000,20000000,21000000,22000000,23000000,24000000,25000000,26000000,27000000,28000000,29000000,30000000,31000000,32000000,33000000,34000000,35000000,36000000,37000000,38000000,39000000,40000000,41000000,42000000,43000000,44000000,45000000,46000000,47000000,48000000,49000000,50000000,51000000,52000000,53000000,54000000,55000000,56000000,57000000,58000000,59000000,60000000,61000000,62000000,63000000,64000000,65000000,66000000,67000000,68000000,69000000,70000000,71000000,72000000,73000000,74000000,75000000,76000000,77000000,78000000,79000000,80000000,81000000,82000000,83000000,84000000,85000000,86000000,87000000,88000000,89000000,90000000,91000000,92000000,93000000,94000000,95000000,96000000,97000000,98000000,99000000,100000000,101000000,102000000,103000000,104000000,105000000,106000000,107000000,108000000,109000000,110000000,111000000,112000000,113000000,114000000,115000000,116000000,117000000,118000000,119000000,120000000,121000000,122000000,123000000,124000000,125000000,126000000,127000000,128000000,129000000,130000000,131000000,132000000,133000000,134000000,135000000,136000000,137000000,138000000,139000000,140000000,141000000,142000000,143000000,144000000,145000000,146000000,147000000,148000000,149000000,150000000,151000000,152000000
7,Count,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2,0,0,0,0,0,0,14,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0

	</pre>
	<pre>
		<img src="http://chart.apis.google.com/chart?cht=lc&chs=800x300&chxt=y,y,x,x&chxp=1,50.0|3,50.0&chtt=Changes+histogram%3A+11&chxr=0,0.0,6.0|1,0.0,100.0|3,0.0,100.0&chd=e:AA..AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAVHAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA..gAAAKPAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA&chxl=1:|Changes%2F1000Kb|2:|500000||||||||||16000000||||||||||31500000||||||||||46500000||||||||||61500000||||||||||77000000||||||||||92500000||||||||||107500000|||||||||121000000|3:|Position"><br>
		11, Position,0,1000000,2000000,3000000,4000000,5000000,6000000,7000000,8000000,9000000,10000000,11000000,12000000,13000000,14000000,15000000,16000000,17000000,18000000,19000000,20000000,21000000,22000000,23000000,24000000,25000000,26000000,27000000,28000000,29000000,30000000,31000000,32000000,33000000,34000000,35000000,36000000,37000000,38000000,39000000,40000000,41000000,42000000,43000000,44000000,45000000,46000000,47000000,48000000,49000000,50000000,51000000,52000000,53000000,54000000,55000000,56000000,57000000,58000000,59000000,60000000,61000000,62000000,63000000,64000000,65000000,66000000,67000000,68000000,69000000,70000000,71000000,72000000,73000000,74000000,75000000,76000000,77000000,78000000,79000000,80000000,81000000,82000000,83000000,84000000,85000000,86000000,87000000,88000000,89000000,90000000,91000000,92000000,93000000,94000000,95000000,96000000,97000000,98000000,99000000,100000000,101000000,102000000,103000000,104000000,105000000,106000000,107000000,108000000,109000000,110000000,111000000,112000000,113000000,114000000,115000000,116000000,117000000,118000000,119000000,120000000,121000000
11,Count,0,0,0,12,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,5,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,6,7,0,0,0,3,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0

	</pre>
</center>

<!--==========================================================================
	Changes by gene 
	========================================================================== -->

<center>
	<hr>
	<b> Details by gene </b><p>
	<p>
	<b><a href="snpEff_genes.txt">Here</a></b> you can find a tab-separated table.
</center>
</div>
</center>